acs-integration-tests/src/test/java/com/ge/predix/integration/test/ACSMeteringIT.java

```diff
@@ -101,37 +106,49 @@ public void setup() throws JsonParseException, JsonMappingException, IOException
         // createNuregoSubscription(this.createPrimaryTestZone.getSubdomain());
     }
 
+    @Test
     public void testACSMetering() throws Exception {
         String testPolicyName = null;
         PolicyEvaluationRequestV1 policyEvaluationRequest = this.policyHelper
                 .createEvalRequest(MARISSA_V1.getSubjectIdentifier(), "sanramon");
         try {
             // Get meter readings before
-            Double beforePolicyUpdateMeterCount = getEntitlementUsageByFeatureId(POLICY_UPDATE_FEATURE_ID, this.zoneId);
-            Double beforePolicyEvalMeterCount = getEntitlementUsageByFeatureId(POLICY_EVAL_FEATURE_ID, this.zoneId);
+            Double beforePolicyUpdateMeterCount = getSteadyEntitlementUsage(POLICY_UPDATE_FEATURE_ID, this.zoneId);
+            Double beforePolicyEvalMeterCount = getSteadyEntitlementUsage(POLICY_EVAL_FEATURE_ID, this.zoneId);
 
-            System.out.println("POLICY UPDATE USAGE BEFORE:" + beforePolicyUpdateMeterCount);
-            System.out.println("POLICY EVAL USAGE BEFORE:" + beforePolicyEvalMeterCount);
+            LOGGER.info("POLICY UPDATE USAGE BEFORE:" + beforePolicyUpdateMeterCount);
+            LOGGER.info("POLICY EVAL USAGE BEFORE:" + beforePolicyEvalMeterCount);
 
             OAuth2RestTemplate acsRestTemplate = this.acsRestTemplateFactory.getACSTemplateWithPolicyScope();
             String policyFile = "src/test/resources/single-site-based-policy-set.json";
             testPolicyName = this.policyHelper.setTestPolicy(acsRestTemplate, getZoneHeaders(), this.zoneUrl,
-                    policyFile);
+                        policyFile);
             ResponseEntity<PolicyEvaluationResult> evalResponse = acsRestTemplate.postForEntity(
                     this.zoneUrl + PolicyHelper.ACS_POLICY_EVAL_API_PATH,
                     new HttpEntity<>(policyEvaluationRequest, getZoneHeaders()), PolicyEvaluationResult.class);
 
             Assert.assertEquals(evalResponse.getStatusCode(), HttpStatus.OK);
 
+            Double afterPolicyUpdateMeterCount = 0.0;
+            Double afterPolicyEvalMeterCount = 0.0;
+
             //Nurego server seems to have a lag before the counts are updated
-            Thread.sleep(3000);
+            for(int i = 0; i < MAX_ITERATIONS; i++){
+                Thread.sleep(10000);
+
+                afterPolicyUpdateMeterCount = getEntitlementUsageByFeatureId(POLICY_UPDATE_FEATURE_ID, this.zoneId);
+                afterPolicyEvalMeterCount = getEntitlementUsageByFeatureId(POLICY_EVAL_FEATURE_ID, this.zoneId);
 
-            Double afterPolicyUpdateMeterCount = getEntitlementUsageByFeatureId(POLICY_UPDATE_FEATURE_ID, this.zoneId);
-            Double afterPolicyEvalMeterCount = getEntitlementUsageByFeatureId(POLICY_EVAL_FEATURE_ID, this.zoneId);
+                LOGGER.info("POLICY UPDATE USAGE AFTER:" + afterPolicyUpdateMeterCount);
+                LOGGER.info("POLICY EVAL USAGE AFTER:" + afterPolicyEvalMeterCount);
```

Comment:
I can see how these log statements were useful while troubleshooting the issue at hand but now they will only serve to clutter the test output. Consider making them debug level.